src/components/screens/SendScreen/screens/TransactionProcessingScreen.tsx

```diff
@@ -0,0 +1,330 @@
+  tokenValue: string;
+  address: string;
+  slicedAddress: string;
+};
+
+const TransactionDetailsBottomSheet: React.FC<
+  TransactionDetailsBottomSheetProps
+> = ({ selectedBalance, tokenValue, address, slicedAddress }) => {
+  const { themeColors } = useColors();
+
+  // Get current date and time for the transaction
+  const now = new Date();
+  const formattedDate = now.toLocaleDateString("en-US", {
+    month: "short",
+    day: "numeric",
+    year: "numeric",
+  });
+  const formattedTime = now
+    .toLocaleTimeString("en-US", {
+      hour: "numeric",
+      minute: "2-digit",
+      hour12: true,
+    })
+    .toLowerCase();
+
+  const dateTimeDisplay = `${formattedDate} · ${formattedTime}`;
+
+  const handleViewOnExplorer = () => {
+    // In the future, this could open a web link to stellar.expert with the transaction details
+    console.log("View on stellar.expert");
+  };
+
+  return (
+    <View className="gap-[24px]">
+      {/* First section with transaction info */}
+      <View className="flex-row gap-[16px]">
+        {selectedBalance && <AssetIcon token={selectedBalance} size="lg" />}
+        <View>
+          <Text md medium primary>
+            {`Sent ${selectedBalance?.tokenCode}`}
+          </Text>
+          <View className="flex-row items-center gap-[4px]">
+            <Icon.ArrowCircleUp size={16} color={themeColors.text.secondary} />
+            <Text sm medium secondary>
+              {dateTimeDisplay}
+            </Text>
+          </View>
+        </View>
+      </View>
+
+      {/* Gray box with transaction data */}
+      <View className="bg-background-secondary rounded-[16px] p-[24px] gap-[12px]">
+        {/* First row: Token amount and Asset icon */}
+        <View className="flex-row items-center justify-between">
+          <View>
+            <Text xl medium primary>
+              {formatAssetAmount(tokenValue, selectedBalance?.tokenCode)}
+            </Text>
+            <Text md medium secondary>
+              {formatFiatAmount(
+                new BigNumber(tokenValue).times(
+                  selectedBalance?.currentPrice || 0,
+                ),
+              )}
+            </Text>
+          </View>
+          {selectedBalance && <AssetIcon token={selectedBalance} size="lg" />}
+        </View>
+
+        {/* Second row: Centered circle with chevron */}
+        <View>
+          <View className="w-[32px] h-[32px] rounded-full bg-tertiary justify-center items-center border border-gray-6">
+            <Icon.ChevronDownDouble
+              size={20}
+              color={themeColors.foreground.primary}
+            />
+          </View>
+        </View>
+
+        {/* Third row: Recipient address info and avatar */}
+        <View className="flex-row items-center justify-between">
+          <View>
+            <Text xl medium primary>
+              {slicedAddress}
+            </Text>
+            <Text md medium secondary>
+              First time send
```

Comment:
wrap with translations hook

Reply:
also, I think this "First time send" feature is low priority, we can leave it out from the MVP if needed